library.gradle

```diff
@@ -22,11 +25,14 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         consumerProguardFiles "consumer-rules.pro"
     }
+
+    // TODO remove after BC5 is merged
     sourceSets {
-        main.java.srcDirs += 'src/main/kotlin'
-        main.java.srcDirs += 'src/main/java'
-        test.java.srcDirs += 'src/test/java'
+        unityIAP {
+            java.srcDirs = ['src/latestDependencies/java']
```

Comment:
I probably should have mentioned this early and it's NABD, but I wonder if we should move all the BC4 code to the `unityIAP` flavor, instead of keeping it in `latestDependencies` and have this be the other way around (`latestDependencies` bringing code from `unityIAP`). That way, once BC5 is ready, it's just moving `bc5SupportDev` to `latestDependencies` whereas the way it is now, we need to move `latestDependencies` to `unityIAP` and then `bc5SupportDev` to `latestDependencies`. 

Having said that, I don't think it's a big deal and we can continue with this.

Reply:
ah yeah, that makes things a little easier, i like it. will ping here once done for re-review..